src/models/Show.ts

```diff
@@ -34,33 +36,41 @@ export default class Show extends Serializable<Show> {
       showJson.field = new Field(showJson.field);
     }
     if (showJson.stuntSheets !== undefined) {
-      showJson.stuntSheets = showJson.stuntSheets.map((stuntSheet: StuntSheet) => {
-        return new StuntSheet(stuntSheet);
+      showJson.stuntSheets.forEach((
+        stuntSheet: StuntSheet,
+        index: number,
+        array: StuntSheet[]
+      ): void => {
+        array[index] = new StuntSheet(stuntSheet);
       });
     }
     this.fromJson(showJson);
   }
 
   /**
-   * For each StuntSheetDot in the specified StuntSheet, calculate and store the flow based on it's continuities in stuntSheetDot.cachedFlow.
+   * For each StuntSheetDot in the specified StuntSheet, calculate and store
+   * the flow based on it's continuities in stuntSheetDot.cachedFlow.
    */
   generateFlows(stuntSheetIndex: number): void {
     if (stuntSheetIndex < 0 || stuntSheetIndex + 1 >= this.stuntSheets.length) {
-      throw `stuntSheetIndex (${stuntSheetIndex}) is invalid with stuntsheet length ${this.stuntSheets.length}`;
```

Comment:
Will the newline here cause the thrown string to have a newline in it?

Reply:
Good catch!